documentation/src/main/asciidoc/userguide/chapters/pc/PersistenceContext.adoc

```diff
@@ -229,7 +229,7 @@ strategy so that we can load entities from the second-level cache, therefore ski
 `withBatchSize(int batchSize)`::
 This setting allows you to specify a batch size for loading the entities (e.g. how many at a time).
 +
-The default is to use a batch sizing strategy defined by the `Dialect.getDefaultBatchLoadSizingStrategy()` method.
+The default is to use a batch sizing strategy defined by the configuration settings in Hibernate. 
```

Comment:
Which configuration settings?

Reply:
Well, I do not have much knowledge about Hibernate-orm. I revise the sentences as follows. This is my best. Please, check if the following contents are correct:
=============
The default is to use a batch sizing strategy defined by the configuration or implementation-specific logic, such as:
In hibernate.cfg.xml: <property name="hibernate.default_batch_fetch_size">16</property>
In Java code: configuration.setProperty("hibernate.default_batch_fetch_size", "16");
In annotation: @BatchSize(size = 16)
=============

Reply:
If you just mean `hibernate.default_batch_fetch_size` then that's a batch size, not a batch sizing strategy.

Reply:
Thank you for checking this. It is certain that the Dialect.getDefaultBatchLoadSizingStrategy()` method was deleted. However, via searching webs, it is difficult for me to find a solution that replace the method. I think that you would just remove the sentence, or ignore this change at this moment.